examples/demo.css

```diff
@@ -256,3 +255,10 @@ a.name-header{
     align-content: center;
     justify-content: center;
 }
+.panel-footer .insert-step{
+  margin-left:10px;
+  margin-top:-40px;
```

Comment:
The padding looked a bit odd, do you know why, when compared to the standard panel shown at https://github.com/publiclab/image-sequencer/issues/673 ?